library/options.inc.php

```diff
@@ -2064,7 +2078,7 @@ function generate_display_field($frow, $currvalue)
 
     // generic selection list or the generic selection list with add on the fly
     // feature
-    if ($data_type == 1 || $data_type == 26 || $data_type == 33) {
+    if ($data_type == 1 || $data_type == 26 || $data_type == 33 || $data_type == 43) {
```

Comment:
also need to use same strategy as above line for the `generate_print_field()` and `generate_plaintext_field()` function in this library